index.html

```diff
@@ -1,289 +1,307 @@
+            </ul>
+        </div>
+    </section>
+    <footer class="contactform">
         <div>
-           <h2>I'm always interested in hearing about new projects, so if you'd like to
-                chat please get in touch.</h2>
-               </div>
-           
-        <div class="formato">
+            <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+        </div>
+
+        <div id="formato">
             <form action="https://formspree.io/f/mwkyoykw" method="post">
                 <ul>
                     <li>
-                        <input type="text" id="name" name="username" required minlength="4" maxlength="30" size="43"
-                            placeholder="Full Name">
+
+                        <input type="text" id="name" name="username" required minlength="4" maxlength="30" size="43" placeholder="Full Name">
                     </li>
-        
+
                     <li>
-                        <input type="email" id="mail" name="user_email" required minlength="4" maxlength="30" size="43"
-                            placeholder="Email address">
+                        <input type="email" id="mail" name="user_email" required minlength="4" maxlength="30" size="43" placeholder="Email address">
                     </li>
-        
+
                     <li>
-                        <textarea id="msg" name="user_message" required maxlength="500"
-                            placeholder="Hello Roger.Glad to see you !"></textarea>
-        
+                        <textarea id="msg" name="user_message" required maxlength="500" placeholder="Hello Roger.Glad to see you !"></textarea>
+
                     </li>
                     <li>
                         <button type="submit" class="get2">Get in touch</button>
                     </li>
-        
+
                 </ul>
             </form>
-        
+
         </div>
         <ul class="palet2">
-            <li><a href="#"><img class="gith" src="pictures/vgit.png" alt="github icon"></a></li>
-            <li> <a href="#"><img src="pictures/vlinkedin.png" alt="github icon"></a></li>
-            <li><a href="#"><img src="pictures/vangelist.png" alt="github icon"></a></li>
-            <li><a href="#"><img src="pictures/vframe.png" alt="github icon"></a></li>
-            <li> <a href="#"><img src="pictures/vmagic.png" alt="github icon"></a></li>
+            <li>
+                <a href="#"><img class="gith" src="pictures/vgit.png" alt="github icon"></a>
+            </li>
+            <li>
+                <a href="#"><img src="pictures/vlinkedin.png" alt=""></a>
+            </li>
+            <li>
+                <a href="#"><img src="pictures/vangelist.png" alt=""></a>
+            </li>
+            <li>
+                <a href="#"><img src="pictures/vframe.png" alt=""></a>
+            </li>
+            <li>
+                <a href="#"><img src="pictures/vmagic.png" alt=""></a>
```

Comment:
- [ ] Kindly add an alt for each image in the document.

Reply:
oh i forget about that sorry going top fixed now